internal/util/paramtable/global_param.go

```diff
@@ -189,17 +143,23 @@ func (p *msgChannelConfig) init(bp *BaseParamTable) {
 	p.initDataCoordTimeTick()
 	p.initDataCoordSegmentInfo()
 	p.initDataCoordSubName()
+
+	p.initDefaultPartitionName()
+	p.initDefaultIndexName()
+	p.initRetentionDuration()
+
+	p.initSimdType()
 }
 
-func (p *msgChannelConfig) initClusterPrefix() {
-	str, err := p.BaseParams.Load("msgChannel.chanNamePrefix.cluster")
+func (p *commonConfig) initClusterPrefix() {
+	str, err := p.BaseParams.Load("common.chanNamePrefix.cluster")
 	if err != nil {
 		panic(err)
 	}
 	p.ClusterPrefix = str
 }
 
-func (p *msgChannelConfig) initChanNamePrefix(cfg string) string {
+func (p *commonConfig) initChanNamePrefix(cfg string) string {
 	value, err := p.BaseParams.Load(cfg)
```

Comment:
need to take care about the compatibility,
if common.xxx can not be found, need to retry msgchannel.xxx in case user is using the previous config file

Reply:
updated